### DISCUSSION TITLE
Add option to packages to make them platform-specific

### DISCUSSION THREAD

**Discussion by federico-terzi** [espanso/espanso#1166] [Feature requests and Ideas]

**Comment by profy12** [comment]
It could be useful to have one package name with 3 configurations specifics to os : maybe a package-osx.yml, package-windows.xml and package-linux.xml

**Comment by federico-terzi** [comment]
Hi @profy12,

Your proposal could be a very good solution indeed, thanks for the feedback. I will definitely consider it when developing the feature.
